bindings/python/crocoddyl/utils/__init__.py

```diff
@@ -1282,7 +1282,8 @@ def solve(self, init_xs=[], init_us=[], maxiter=100, isFeasible=False, regInit=N
                     self.dV = self.tryStep(a)
                 except ArithmeticError:
                     continue
-                d1, d2 = self.expectedImprovement()
+                self.d = self.expectedImprovement()
+                d1, d2 = np.asscalar(self.d[0]), np.asscalar(self.d[1])
```

Comment:
This will break with newer versions of NumPy (was fixed in https://github.com/cmastalli/crocoddyl/pull/11)

Reply:
I am planning to merge later that commit. However, I have fixed this issue for this particular line.